src/app/components/Footer.tsx

```diff
@@ -8,13 +8,13 @@ export default function Footer() {
       <div className="container">
         <div className="row">
           <div className="col">
-            <a href="https://suffolk.edu">
-              <img
-                src="suffolk_law.png"
-                alt="Suffolk University Law School"
-                className="img-fluid mb-2 suffolk-logo"
-                width={497}
-                height={70}
+            <a href="https://suffolklitlab.org">
+              <Image
```

Comment:
Since we're using Next I thought I would use Next's [image component](https://nextjs.org/docs/app/api-reference/components/image), which seems to have some image optimization advantages.